### PR TITLE
Event Triggers: Take webhook url from env ( close #966)

### LINKED ISSUE #966: Event Triggers: Webhook Url should also take an env var
This will help in keeping same migrations for different environments.

@dsandip Will need console changes.

### PR DESCRIPTION
**Pull request by tirumaraiselvan**
### Description

What component does this PR affect? 

- [x] Server
- [ ] Console
- [ ] CLI
- [ ] Docs
- [ ] Community Content
- [ ] Build System

Requires changes from other components? If yes, please mark the components:

- [ ] Server
- [x] Console
- [ ] CLI
- [x] Docs
- [ ] Community Content
- [ ] Build System

### Type

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Docs update
- [ ] Community content

### Checklist:

- [x] I have read the **[contributing guide](https://github.com/hasura/graphql-engine/blob/master/CONTRIBUTING.md)** and my code conforms to the guidelines.
- [ ] This change requires a change in the documentation. 
- [ ] I have updated the documentation accordingly.
- [ ] I have added required tests.

### CODE REVIEW

**Comment by tirumaraiselvan** [comment]
@dsandip @karthikvt26 Pls add the console changes.

File: server/src-lib/Hasura/RQL/Types/Subscribe.hs [hunk 1]
```diff
@@ -169,16 +193,44 @@ data EventTrigger
...
+    webhookConf <- case (webhook, webhookFromEnv) of
+                        (Just w, Nothing)  -> return $ WCValue w
+                        (Nothing, Just wEnv) -> return $ WCEnv wEnv
+                        (Just _, Just _) -> fail "only one of webhook or webhook_from_env should be given"
+                        _ ->   fail "must provide webhook or webhook_from_env"
+    return $ EventTriggerConf name opsDef webhookConf retryConf headers
+  parseJSON _ = fail "expecting object for event_trigger_def"
+
+instance ToJSON EventTriggerConf where
+  toJSON (EventTriggerConf name def (WCValue w) rc headers) = object ["name" .= name
```
Comment (0x777) [inline server/src-lib/Hasura/RQL/Types/Subscribe.hs]: indentation

**Comment by shahidhk** [comment]
@rikinsk Can we add the docs for this, since Tiru is on leave?

**Pull request merged**